.github/workflows/ci-cd.yml

```diff
@@ -51,11 +51,9 @@ jobs:
           yarn nx affected --target=lint --base=${{ env.BASE_BRANCH }} --parallel=3
       - name: build & test
         run: |
-          export VITE_RTS_STREAM_NAME=${{ secrets.STREAM_NAME }}
           eval "yarn nx launch-server e2e-test --skip-nx-cache"
-          eval "yarn nx affected --target=e2e --base=${{ env.BASE_BRANCH }} --parallel=1"
+          eval "yarn nx affected --target=e2e --base=${{ env.BASE_BRANCH }} --parallel=1 --skip-nx-cache"
           eval "yarn nx publisher-viewer e2e-test --skip-nx-cache --report=./apps/e2e-test/reports/publisher-viewer/"
```

Comment:
Is that possible we move the test scenarios into `publisher` and `viewer` app? So the `affected` parameter could detect if it should run the specific test for the change in different app.

Reply:
Fixed